examples/ibm-iks-openshift/README.md

```diff
@@ -1,24 +1,141 @@
+
+```hcl
+resource "ibm_container_cluster" "testacc_cluster" {
+  name            = "test"
+  datacenter      = "dal10"
+  machine_type    = "free"
+  hardware        = "shared"
+  public_vlan_id  = "vlan"
+  private_vlan_id = "vlan"
+  subnet_id       = ["1154643"]
+
+  default_pool_size = 1
+
+  webhook {
+    level = "Normal"
+    type  = "slack"
+    url   = "https://hooks.slack.com/services/yt7rebjhgh2r4rd44fjk"
+  }
+}
 ```
 
-For apply phase
+Create the Kubernetes cluster with a default worker pool with 2 workers and one standalone worker as mentioned by worker_num:
+
+```hcl
+resource "ibm_container_cluster" "testacc_cluster" {
+  name            = "test"
+  datacenter      = "dal10"
+  machine_type    = "free"
+  hardware        = "shared"
+  public_vlan_id  = "vlan"
+  private_vlan_id = "vlan"
+  subnet_id       = ["1154643"]
 
-```shell
-terraform apply
+  default_pool_size = 2
+  worker_num        = 1
+  webhook {
+    level = "Normal"
+    type  = "slack"
+    url   = "https://hooks.slack.com/services/yt7rebjhgh2r4rd44fjk"
+  }
+}
 ```
 
-For destroy
+Create a Gateway Enabled Kubernetes cluster:
 
-```shell
-terraform destroy
+```hcl
+resource "ibm_container_cluster" "testacc_cluster" {
+  name            = "testgate"
+  gateway_enabled = true 
+  datacenter      = "dal10"
+  machine_type    = "b3c.4x16"
+  hardware        = "shared"
+  private_vlan_id = "2709721"
+  private_service_endpoint = true
+  no_subnet = false
+}
 ```
+
+```hcl
+
+data "ibm_container_cluster" "cluster_foo" {
+  cluster_name_id = "FOO"
+}
+
+```
+
+## Examples
+
+* [ Container cluster ](https://github.com/IBM-Cloud/terraform-provider-ibm/tree/master/examples/ibm-iks-openshift)
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | ~> 0.12 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| ibm | n/a |
+
+## Inputs
+
+| Name | Description | Type | Required |
+|------|-------------|------|---------|
+| name | Name of the cluster. | `string` | yes |
+| datacenter | The datacenter of the worker nodes. | `string` | yes |
+| machine\_type | The machine type of the worker nodes. | `string` | no |
+| hardware | The level of hardware isolation for your worker node.  | `string` | no |
+| public\_vlan_id | The public VLAN ID for the worker node. | `string` | no |
+| private\_vlan_id | The private VLAN of the worker node. | `string` | no |
+| subnet\_id | The existing subnet ID that you want to add to the cluster.| `string` | no |
+| default\_pool\_size | The number of workers created under the default worker pool which support Multi-AZ.| `int` | no |
+| webhook | The webhook that you want to add to the cluster. | `string` | no |
+| worker\_num | The number of cluster worker nodes. This creates the stand-alone workers which are not associated to any pool.| `int` | no |
+| cluster_name_id | The name or ID of the cluster. | `string` | yes |
+
+
+## Outputs
+
+| Name | Description |
```

Comment:
I think the output from this example is cluster_config_file_path not id,name,subnet_id